german/Korallen/Hoehle/hoehle.md

```diff
@@ -5,4 +5,6 @@ Im Inneren angekommen...
 
 [Treffen Sie auf eine Meerjungfrau](Meerjungfrau/meerjungfrau.md)
 
-[Entdecken Sie eine Luftblasen, tauchen auf und forschen weiter](Luftblase/luftblase.md)
\ No newline at end of file
+[Entdecken Sie eine Luftblasen, tauchen auf und forschen weiter](Luftblase/luftblase.md)
+
+[Entdecken Sie den Lichtschein einer geheimen Unterwasserstation](Unterwasserstation/Unterwasserstation.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.